### PR TITLE
Docker: Add env vars to compose files

### PR DESCRIPTION
**Pull request by stephendade**
To aid user configuration, I've added the required environmental vars to reference the server's external IP address.

I've also added some other environmental vars that a user can uncomment and use if needed.

### CODE REVIEW

**Review by brothercorvo — APPROVED** [review]
@RadioAndrea this seems reasonable, a part that all the default IPs should be set to 0.0.0.0

**Comment by RadioAndrea** [comment]
Performed a manual merge, most of this was duplicate of work we already had in the container working branch, but I did include a few of the wordings.

I don't want to recommend people to update the IPs at the bottom of the file because we already have documentation about those, and the descriptions in the file could lead people astray if they run multiple container networks or have something like a reverse proxy in place.

File: containers/compose.yaml
Comment (RadioAndrea) [inline containers/compose.yaml:1]: This file was removed in our compose branch

File: containers/example-compose.yaml [hunk 1]
```diff
@@ -37,3 +59,9 @@ services:
     volumes:
       - ./ui-config:/home/freetak/data:rw,Z
     network_mode: "host"
+    environment:
+      # Change the following 3 lines to the FTS external IP
```
Comment (RadioAndrea) [inline containers/example-compose.yaml:63]: the more I think about this the less I like it, I don't think we can conclusively say this for all use cases if the network is even slightly atypical.

File: containers/example-compose.yaml
Comment (RadioAndrea) [inline containers/example-compose.yaml:1]: Unfortunately much of this is very similar to what already existed in our compose working branch

**Review by RadioAndrea — COMMENTED** [review]
Most of this is rework that was already in a branch, but some wording is helpful for user guidance

**Pull request closed without merging**